site/public/css/forum.css

```diff
@@ -230,18 +230,35 @@
     text-decoration: none !important;
 }
 
+
 .post-action-container {
-    margin-top: 8px;
+    margin-top: 5px;
     margin-left: 10px;
     float: right;
+    display: flex;
+    align-items: center;
 }
 
+
 .post-email-toggle,
 .post-user-info {
     margin-right: 2px;
     display: inline-block;
 }
 
+.upduck-container{
```

Comment:
I'm not sure how I feel about the whole span being presented as a "button", especially since only the duck is clickable. I also feel that the design is a bit at odds with the rest of the post actions in terms of design.